examples/mpi/lammps/Dockerfile

```diff
@@ -1,29 +1,41 @@
 # ch-test-scope: full
 FROM openmpi
-WORKDIR /
+WORKDIR /usr/local/src
 
 # Packages for building.
 RUN apt-get install -qy --no-install-recommends \
+    cmake \
     git \
     patch \
-    python-dev
+    python-dev \
+    python-pip \
+    python-setuptools 
 
+# Building mpi4py from source to ensure it is built against our MPI build
+# Building numpy from source to work around issues seen on Aarch64 systems
+RUN pip install --no-binary=mpi4py,numpy mpi4py numpy 
```

Comment:
I think you can say `--no-binary=:all:`? Or are there dependencies where you'll accept binary packages? If the latter, say that briefly.

Reply:
I'll change to `--no-binary :all: mpi4py numpy` as I'm not seeing a behavior difference between the two.